lib/pf/scan/nessus6.pm

```diff
@@ -26,7 +26,7 @@ use pf::util;
 use pf::node;
 use pf::constants qw($TRUE $FALSE);
 use pf::constants::scan qw($SCAN_SECURITY_EVENT_ID $PRE_SCAN_SECURITY_EVENT_ID $POST_SCAN_SECURITY_EVENT_ID $STATUS_STARTED);
-use Net::Nessus::REST;
+use Net::Nessus::REST_M;
```

Comment:
Will this module still work with Nessus 6 ?

Reply:
@julsemaan  Yep , I've just added a routine (In the nessus rest implementation  - check out REST_M)  to extract the x-api token so we can call the create_Scan api